src/net.h

```diff
@@ -1141,6 +1141,12 @@ class CConnman
     friend struct ConnmanTestMsg;
 };
 
+/**
+ * Get a unique identifier for a peer based on the connections network and the
```

Comment:
This might be confusing for the reader:
- function name refers to Network ID
- the comment refers to network + socket address

Reply:
I guess the issue is `Network's ID` vs `peer's ID`